style.css

```diff
@@ -1,4 +1,624 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400&family=Poppins:wght@400;500;700;800&display=swap');
 
-h1 {
-  color: aqua;
+:root {
+  --color-primary: #272a31;
+  --color-white: #fff;
+  --color-black: #000;
+  --color-secondary: #ec5242;
+}
+
+*,
+*::after,
+*::before,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  color: var(--color-white);
+  font-size: 1.7rem;
+  line-height: 1.5;
+  font-weight: 400;
+}
+
+.row-style {
+  width: 88% !important;
+  margin: 0 auto !important;
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  left: 0;
+  background-color: #fff;
+  box-shadow: 0 0.5rem 1rem 0.2rem rgb(0 0 0 / 5%);
+  z-index: 999;
+}
+
+.header-top {
+  display: none;
+}
+
+.header-bottom {
+  display: flex;
+  flex-direction: row-reverse;
+  min-height: 6rem;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header .navbar {
+  display: flex;
+  align-items: center;
+}
+
+.navbar .nav-link {
+  text-decoration: none;
+  color: var(--color-primary);
+  font-size: 1.6rem;
+  transition: all 0.2s;
+  font-weight: 600;
+}
+
+.logo-image {
+  display: none;
+}
+
+.main {
+  overflow: hidden;
+}
+
+.section-hero {
+  background-image:
+    linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
+    url(images/hero-background.jpg);
+  min-height: calc(100vh - 9rem);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 4rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.hero-title-text {
+  color: var(--color-secondary);
+  font-size: 2.2rem;
+}
+
+.hero-text {
+  color: #d3d3d3;
+  font-size: 2.4rem;
+}
+
+.hero-heading {
+  font-size: 4rem;
+  font-weight: 700;
+  color: var(--color-secondary);
+}
+
+.margin-medium {
+  margin-top: 3rem !important;
+  margin-bottom: 1.5rem;
+}
+
+.event-details {
+  margin: 3rem auto;
+}
+
+.event-details a {
+  color: #d3d3d3;
+}
+
+hr {
+  width: 10rem;
+  border: 2px solid var(--color-secondary);
+}
+
+#special-button a {
+  color: var(--color-secondary);
+  border: 2px var(--color-secondary) solid;
+  padding: 0.5rem 1.5rem;
+  display: inline;
+}
+
+.header-buttons {
+  display: flex;
+  gap: 2rem;
+}
+
+.hero-button {
+  padding: 0.6rem 1.8rem;
+  border-radius: 0.3rem;
+  font-size: 1.6rem;
+  letter-spacing: 0.2rem;
+}
+
+.attend-button {
+  background-color: #ec5242;
+  border: solid 2px #ec5242;
+  color: #fff;
+  transition: all 0.2s;
+}
+
+.sponsor-button {
+  background-color: transparent;
+  border: solid 2px #fff;
+  color: #fff;
+  transition: all 0.2s;
+}
+
+.section-program {
+  background-color: #f7f7f7;
+  background-image: url(images/hero-bg.png);
+  min-height: 40rem;
+  padding-top: 6rem;
+  padding-bottom: 4rem;
+}
```

Comment:
- [ ] [OPTIONAL] I appreciate your hard work 👍, but the design of the cards in the `"Program"` section is not the same as the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in the mobile version, and there is no `"Join"` button, so it would be great if you could design this section like [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 😃 

![image](https://user-images.githubusercontent.com/63915024/198380163-4bb62539-b4f3-4cad-a842-32ce38758bce.png)